tests/rptest/clients/ping_pong.py

```diff
@@ -0,0 +1,197 @@
+from confluent_kafka import Producer, Consumer, KafkaException, KafkaError
+from confluent_kafka import TopicPartition, OFFSET_BEGINNING
+import random
+
+import time
+
+
+class SyncProducer:
+    def __init__(self, bootstrap):
+        self.bootstrap = bootstrap
+        self.producer = None
+        self.last_msg = None
+
+    def init(self):
+        self.producer = Producer({
+            "bootstrap.servers": self.bootstrap,
+            "topic.metadata.refresh.interval.ms": 5000,
```

Comment:
There are a lot of config properties being set here -- I would rather test that a "normal" client (i.e. mostly using defaults) will work.

Reply:
The problem is that "normal" client won't work in this situation. With the default properties the client may hang for quite a while, I set all time related properties to make sure the client returns control flow as soon as possible.

Reply:
I stole the config from chaos tests where it gradually evolved into this to avoid uncontrolled pauses but here it is indeed excessive, changed to default. Thanks for pointing it!